### PR TITLE
Enable TRTorch compilation with TensorRT 8

### PR DESCRIPTION
**Pull request by peri044**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by narendasan** [comment]
GA is not out yet right @peri044?

File: cpp/trtorchexec/main.cpp [hunk 1]
```diff
@@ -57,12 +57,13 @@ int main(int argc, const char* argv[]) {
...
   auto compile_spec = trtorch::CompileSpec(dims);
   compile_spec.workspace_size = 1 << 24;
+  compile_spec.sparse_weights = true;
 
-  std::cout << "Checking operator support" << std::endl;
-  if (!trtorch::CheckMethodOperatorSupport(mod, "forward")) {
-    std::cerr << "Method is not currently supported by TRTorch" << std::endl;
-    return -1;
-  }
+  // std::cout << "Checking operator support" << std::endl;
```
Comment (narendasan) [inline cpp/trtorchexec/main.cpp:62]: Why is this commented out?

File: WORKSPACE [hunk 2]
```diff
@@ -70,20 +70,20 @@ http_archive(
...
 http_archive(
     name = "tensorrt",
     build_file = "@//third_party/tensorrt/archive:BUILD",
-    sha256 = "d3a1f478e304b48878604fac70ce7920fece71f9cac62f925c9c59c197f5d087",
-    strip_prefix = "TensorRT-7.2.3.4",
+    sha256 = "def6a5ee50bed25a68a9c9e22ec671a8f29ee5414bde47c5767bd279e5596f88",
+    strip_prefix = "TensorRT-8.0.1.6",
     urls = [
-        "https://developer.nvidia.com/compute/machine-learning/tensorrt/secure/7.2.3/tars/TensorRT-7.2.3.4.Ubuntu-18.04.x86_64-gnu.cuda-11.1.cudnn8.1.tar.gz",
+        "https://developer.nvidia.com/compute/machine-learning/tensorrt/secure/8.0.1/tars/tensorrt-8.0.1.6.linux.x86_64-gnu.cuda-11.3.cudnn8.2.tar.gz",
```
[the remark this answers is not in the document]
Comment (peri044) [reply WORKSPACE:86]: We have only single TensorRT tar for cuda-11.1/2/3. So TRT is compatible with all three. 
Pytorch 1.9 is using 11.1 atleast from the installer page.

File: cpp/trtorchexec/main.cpp (hunk 1 above)
Comment (peri044) [reply cpp/trtorchexec/main.cpp:62]: This file shouldn't have any changes. Will fix now.

Comment (peri044) [reply cpp/trtorchexec/main.cpp]: done

**Pull request closed without merging**